opendps/command_handler.c

```diff
@@ -107,20 +141,27 @@ static void on_cmd(uint32_t argc, char *argv[])
 {
     (void) argc;
     (void) argv;
-    printf("Power on\n");
+    printf("Power on\r\n");
     pwrctl_enable_vout(true);
     ui_update_power_status(true);
+
+
+#warning TEMPORARY HACK TO UNFREEZE LCD
```

Comment:
This should go with an ifdef UNFREEZE_LDC_HACK coming from dps-model.h and of course need to be set for the 5005s.